media/stylus/includes/mixins_indicators.styl

```diff
@@ -0,0 +1,136 @@
+/*
+Mixins for indicators
+====================================================================== */
+$icon-spacing = 30px;
+$inline-horizontal-padding = 0.35em;
+
+/*
+block indicators aka banners
+-------------------------------------------------------------- */
+
+$indicator-block {
+    set-message-base(true);
+    clearfix();
+
+    background-color: $light-background-color;
+    font-size: $smaller-font-size;
+
+    &:before {
+        display: inline-block;
+        width: 30px;
+        bidi-value(float, left, right);
+        bidi-value(margin, -5px 0 -5px ($icon-spacing * -1), -5px ($icon-spacing * -1) -5px 0);
+        font-family: "FontAwesome";
+        font-size: $larger-font-size;
+        font-style: normal;
+        font-weight: normal;
+        text-align: center;
+    }
+
+    pre {
+        color: $text-color;
+    }
+}
+
+$indicator-block-heading {
+    font-size: $base-font-size;
+    font-weight: bold;
+}
+
+$indicator-system {
+    border-width: 2px
+    border-style: solid;
+    font-size: $base-font-size;
+}
+
+/*
+inline indicators aka badges
+-------------------------------------------------------------- */
+
+$indicator-inline {
+    display: inline-block;
+    min-width: 20px;
+    vertical-align: baseline;
+    bidi-style(margin-left, ($grid-spacing / 2 ), margin-right, 0);
+    border-radius: 2px;
+    bidi-style(border-left-width, 4px, border-right-width, 0);
+    bidi-style(border-left-style, solid, border-right-style, none);
```

Comment:
I had to read the spec to see if colour was optional in the short hand declaration but, yes, yes I can. 💫